### DISCUSSION TITLE
Can I use the G7 pre-configured with QR version of xdrip to use with a G6?

### DISCUSSION THREAD

**Discussion by FlyingToto** [NightscoutFoundation/xDrip#2995] [Q&A] [answered]
So I am currently happily running xdrip with G7 with AAPS 3.1 by following the instruction with the QR code here:  https://navid200.github.io/xDrip/docs/Dexcom/G7.html 

Now I want to switch back to a G6 sensor.  do I need to do anything special outside of disabling smoothing and registering my G6 transmitter/sensor? 

thanks,

**Comment by Navid200** [comment, marked as answer]
As far as setting up xDrip, all you need to do is:
1- If you are using an older version of xDrip, [update](https://navid200.github.io/xDrip/docs/Updates.html) to the current stable release.
2- Change the transmitter ID to that of G6.
3- [Activate](https://navid200.github.io/xDrip/docs/Dexcom/NewG6TX_Activation.html) the G6 transmitter.
4- Wait for the status page to be populated with the battery voltages, and approve a pair request while waiting.
5- Start sensor.

As far as what you need to do for AAPS, please ask that team.

**Reply by FlyingToto** [reply]
Thank you so much! that's what I was hoping for: (ie: xdrip code is actually the same between G6/G7 it is just and there are no other secret settings required for G7)
